src/app/pages/applications/forms/kubernetes-settings.component.ts

```diff
@@ -13,7 +13,7 @@ import { ApplicationsService } from '../applications.service';
 })
 export class KubernetesSettingsComponent {
   protected queryCall: string = 'kubernetes.config';
-  protected editCall: string = 'kubernetes.update';
+  protected editCallJob: string = 'kubernetes.update';
```

Comment:
Not familiar with UI code but wonder if the job would make more sense to be a boolean so the editCall parameter is consistent everywhere

Reply:
That make sence.